src/coreclr/jit/emitarm64.cpp

```diff
@@ -29338,6 +29938,165 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insLatency    = PERFSCORE_LATENCY_2C;
             break;
 
+        case IF_SVE_HY_3A: // .........h.mmmmm ...gggnnnnn.oooo -- SVE 32-bit gather prefetch (scalar plus 32-bit scaled
```

Comment:
These are all 1C/1C and can be squashed into a single case

Reply:
We have quite a few SVE cases that can be squashed. Should we enforce cleaning these up proactively as we add encodings, or should we expect to do this cleanup later? I'm fine either way.

Reply:
Since these values are not right and need to be updated anyway, I tend not to squash them as the instructions could be different when we go back and fix these up.